provisioning/plugins/common/unsignedcorim_decoder.go

```diff
@@ -0,0 +1,91 @@
+package common
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/corim/corim"
+	"github.com/veraison/endorsement"
+	"github.com/veraison/veraison/provisioning/decoder"
+)
+
```

Comment:
Perhaps a comment on Line 31 would help, explaining the purpose and what implementations are currently supported and what will be done in future?

Reply:
I am a bit confused, are you sure it's L31?

Reply:
Sorry I meant Line 13

Reply:
ah ok!  sure

Reply:
Done in 17c2534